docs/presets/neutrino-lint-base/README.md

```diff
@@ -0,0 +1,179 @@
+# Neutrino Lint Base
+[![NPM version][npm-image]][npm-url] [![NPM downloads][npm-downloads]][npm-url] [![Join Slack][slack-image]][slack-url]
+
+`neutrino-lint-base` is an abstract Neutrino preset base that makes creating ESLint-based presets simpler. By creating
+a linting preset that extends from `neutrino-lint-base`, the development overhead and dependencies needed can be
+significantly reduced. Use it as a baseline to create your own linting presets easier and more quickly.
+
+## Features
+
+- Quickly and easily create your own Neutrino linting presets
+- Modern Babel knowledge supporting ES modules, JSX (when used with React preset), Web and Node.js apps
+- Highly visible during development, fails compilation when building for production
+- Easily extensible to customize your project as needed
+
+## Development Requirements
+
+- Node.js v6.9+
+- Yarn or npm client
+- Neutrino v4
+
+## Installation
+
+`neutrino-lint-base` can be installed via the Yarn or npm clients.
+
+#### Yarn
+
+```bash
+❯ yarn add neutrino-lint-base
```

Comment:
No, the lint base is used in actual preset which is required by the dependent project. Extending from any preset needs it as a dependency.